 12-libedit-Makefile.diff by Anibal Monsalve Salazar
 without this patch I get the following message:
 make: ${_MKTARGET_CREATE} expands to empty string

--- a/libedit/Makefile	2007-05-28 22:06:18.000000000 +1000
+++ a/libedit/Makefile	2008-06-15 13:53:50.000000000 +1000
@@ -59,45 +59,37 @@ CLEANFILES+=	${AHDR} fcns.h help.h fcns.
 SUBDIR=	readline
 
 vi.h: vi.c makelist Makefile
-	${_MKTARGET_CREATE}
 	${HOST_SH} ${LIBEDITDIR}/makelist -h ${LIBEDITDIR}/vi.c \
 	    > ${.TARGET}.tmp && \
 	    mv ${.TARGET}.tmp ${.TARGET}
 
 emacs.h: emacs.c makelist Makefile
-	${_MKTARGET_CREATE}
 	${HOST_SH} ${LIBEDITDIR}/makelist -h ${LIBEDITDIR}/emacs.c \
 	    > ${.TARGET}.tmp && \
 	    mv ${.TARGET}.tmp ${.TARGET}
 
 common.h: common.c makelist Makefile
-	${_MKTARGET_CREATE}
 	${HOST_SH} ${LIBEDITDIR}/makelist -h ${LIBEDITDIR}/common.c \
 	    > ${.TARGET}.tmp && \
 	    mv ${.TARGET}.tmp ${.TARGET}
 
 fcns.h: ${AHDR} makelist Makefile
-	${_MKTARGET_CREATE}
 	${HOST_SH} ${LIBEDITDIR}/makelist -fh ${AHDR} > ${.TARGET}.tmp && \
 	    mv ${.TARGET}.tmp ${.TARGET}
 
 fcns.c: ${AHDR} fcns.h help.h makelist Makefile
-	${_MKTARGET_CREATE}
 	${HOST_SH} ${LIBEDITDIR}/makelist -fc ${AHDR} > ${.TARGET}.tmp && \
 	    mv ${.TARGET}.tmp ${.TARGET}
 
 help.c: ${ASRC} makelist Makefile
-	${_MKTARGET_CREATE}
 	${HOST_SH} ${LIBEDITDIR}/makelist -bc ${ASRC} > ${.TARGET}.tmp && \
 	    mv ${.TARGET}.tmp ${.TARGET}
 
 help.h: ${ASRC} makelist Makefile
-	${_MKTARGET_CREATE}
 	${HOST_SH} ${LIBEDITDIR}/makelist -bh ${ASRC} > ${.TARGET}.tmp && \
 	    mv ${.TARGET}.tmp ${.TARGET}
 
 editline.c: ${OSRCS} makelist Makefile
-	${_MKTARGET_CREATE}
 	${HOST_SH} ${LIBEDITDIR}/makelist -e ${OSRCS:T} > ${.TARGET}.tmp && \
 	    mv ${.TARGET}.tmp ${.TARGET}
 
